system/jlib/jdebug.cpp

```diff
@@ -1966,6 +1977,73 @@ class CExtendedStats  // Disk network and cpu stats
 
         if (!getInterfaceName(ifname))
             ifname.set("eth0");
+
+#ifdef __linux__
+        // MCK - wish libblkid could do this ...
+        // Another way might also be to look for:
+        //   /sys/block/sd*
+        //   /sys/block/nvme*
+        // and match those with entries in /proc/diskstats
+        StringBuffer cmd("lsblk -o TYPE,MAJ:MIN --pairs");
```

Comment:
Is there a cleaner way to get the disk info ?

Reply:
I think reasonable to do this way.